manim/mobject/vector_field.py

```diff
@@ -64,7 +64,7 @@ def __init__(
         color: Optional[Color] = None,
         color_scheme: Callable[
             [np.ndarray], float
-        ] = np.linalg.norm  # TODO maybe other default for direction?
+        ] = np.linalg.norm,  # TODO maybe other default for direction?
```

Comment:
Assigning an external function as a default argument seems to mess with our caching mechanism; you might have to set the default argument to `None` and change it during `__init__` or so.

Reply:
Ok, I have to look into that later

Reply:
Hm, that didn't help :/

Reply:
It's fixed by now